src/ol_infrastructure/applications/mitopen/Pulumi.applications.mitopen.Production.yaml

```diff
@@ -9,44 +9,48 @@ config:
   heroku_app:interpolation_vars:
     auth_allowed_redirect_hosts:
     - "ocw.mit.edu"
-    - "mitopen.odl.mit.edu"
-    session_cookie_domain: ".mitopen.odl.mit.edu"
+    - "mitopen.odl.mit.edu"  # legacy
+    - "learn.mit.edu"
+    session_cookie_domain: ".learn.mit.edu"
     csrf_domains:
-    - "https://api.mitopen.odl.mit.edu"
-    - "https://mitopen.odl.mit.edu"
+    - "https://api.learn.mit.edu"
+    - "https://api.mitopen.odl.mit.edu"  # legacy
+    - "https://learn.mit.edu"
+    - "https://mitopen.odl.mit.edu"  # legacy
     cors_urls:
     - "https://draft.ocw.mit.edu"
-    - "https://live-qa.ocw.mit.edu"
+    - "https://live-qa.ocw.mit.edu"  # Why?
```

Comment:
Because the OCW site uses the Learn app for the search functionality

Reply:
Should have been more specific. Why QA -> Prod?

Reply:
I think because the QA index didn't have any real data in it.